project/plugins.sbt

```diff
@@ -1,6 +1,5 @@
 addSbtPlugin("pl.project13.scala" % "sbt-jmh"        % "0.3.3")
 addSbtPlugin("com.dwijnand"       % "sbt-travisci"   % "1.1.1")
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"    % "0.6.21")
-addSbtPlugin("com.47deg"          % "sbt-microsites" % "0.7.15")
-addSbtPlugin("com.geirsson"       % "sbt-scalafmt"   % "1.4.0")
-addSbtPlugin("com.dwijnand"       % "sbt-dynver"     % "3.0.0")
+addSbtPlugin("org.scala-js"       % "sbt-scalajs"    % "0.6.22")
```

Comment:
https://github.com/sjrd/sbt-dynscalajs is far superior

Reply:
@fommil Are you sure we want this here and we will truly benefit from it? There is a few reasons why this plugin shouldn't be used.

Reply:
We don't have anything to my knowledge that requires the complicated multi-project build that `sbt-scalajs` makes, which is the main (only?) reason to use `sbt-dynscalajs`. That said, that isn't the point of this PR, but it's probably worth making a PR to switch to see what other people think.